.github/workflows/backend-tests.yml

```diff
@@ -42,7 +42,7 @@ jobs:
           JWT_SECRET_KEY: ${{secrets.JWT_SECRET_KEY}}
           CI_TEST_API_KEY: ${{secrets.CI_TEST_API_KEY}} 
         run: |
-          pytest
+          python -m pytest tests/
```

Comment:
pytest doesnt add directory to the path automatically. Python does so this saves on having to export every test.